proto/skaffold.proto

```diff
@@ -547,6 +547,46 @@ enum StatusCode {
 
     // Unable to connect to cluster
     DEPLOY_CLUSTER_CONNECTION_ERR = 1001;
+    // Could not retrieve debug helpers.
+    DEPLOY_DEBUG_HELPER_RETRIEVE_ERR = 1002;
+    // Deploy clean up error
+    DEPLOY_CLEANUP_ERR = 1003;
+    // Unable to apply helm labels.
+    DEPLOY_HELM_APPLY_LABELS = 1004;
+    // Deploy error due to user deploy config for helm deployer
+    DEPLOY_HELM_USER_ERR = 1005;
+    // Helm error when no build result is found of value  specified in helm `artifactOverrides`
+    DEPLOY_NO_MATCHING_BUILD = 1006;
+    // Unable to get helm client version
+    DEPLOY_HELM_VERSION_GET_ERR = 1007;
+    // Helm version not supported. Should be 3.0.0-beta.0 or greater.
+    DEPLOY_HELM_MIN_VERSION_ERR = 1008;
+    // Unable to retrieve kubectl version
+    DEPLOY_KUBECTL_VERSION_ERR = 1109;
+    // User specified offline mode for rendering but remote manifests presents.
+    DEPLOY_KUBECTL_OFFLINE_MODE_ERR =1010;
+    // Error waiting for previous version deletion before next version is active.
+    DEPLOY_ERR_WAITING_FOR_DELETION = 1011;
+    // Error reading manifests
+    DEPLOY_READ_MANIFEST_ERR = 1012;
+    // Error reading remote manifests
+    DEPLOY_READ_REMOTE_MANIFEST_ERR = 1013;
+    // Errors listing manifests
+    DEPLOY_LIST_MANIFEST_ERR = 1014;
+    // Deploy error due to user deploy config for kubectl deployer
+    DEPLOY_KUBECTL_USER_ERR = 1015;
+    // Deploy error due to user deploy config for kustomize deployer
+    DEPLOY_KUSTOMIZE_USER_ERR = 1016;
+    // Error replacing a built artifact in the manifests
+    DEPLOY_REPLACE_IMAGE_ERR = 1017;
+    // Error transforming a manifest during skaffold debug
+    DEPLOY_TRANSFORM_MANIFEST_ERR = 1018;
+    // Error setting user specified additional labels.
+    DEPLOY_SET_LABEL_ERR = 1019;
+    // Error writing hydrated kubernetes manifests.
```

Comment:
Should this be _rendered kubernetes manifests_?  _hydrated_ has a specific meaning elsewhere.